README.md

```diff
@@ -13,8 +13,10 @@ cd plotIt/
 
 # Initialize the git remotes
 source firstsetup.sh 
-# Load the proper environment (a la CMSSW_7_4_10)
-source setup_sl6_env.sh
```

Comment:
Just to be clear:

 - `setup_sl6_env.sh` just load a more recent version of gcc and boost, using `cvmfs`. It's basically equivalement of doing `module load gcc/...` in ingrid, but it also works on lxplus.
- `setup_for_cms_env.sh` setup the env. for compiling plotIt within a CMSSW env. All it does is exporting `BOOST_ROOT` in order for the Makefile to use the correct boost version (the one from CMSSW, not the system one)